com.kms.cura.dal/src/com/kms/cura/dal/database/DatabaseHelper.java

```diff
@@ -140,4 +140,26 @@ public void insertReferenceRowsToReferenceTable(String database, List<ReferenceT
 			}
 		}
 	}
+	
+	public Entity queryEmailPasswordJoin(String tableName1, String tableName2, String email, String password, String onTable1, String onTable2) throws SQLException, ClassNotFoundException{
+		PreparedStatement stmt = null;
+		ResultSet rs = null;
+		try {
+			stmt = con.prepareStatement(
+					"SELECT * FROM " + tableName1 + " LEFT JOIN " + tableName2 + " ON " + tableName1+"."+onTable1 + " = " + tableName2 + "."+ onTable2 +
+					" WHERE " + UserColumn.EMAIL.getColumnName() + " = ? AND "+UserColumn.PASSWORD.getColumnName() + " = ?");
+			stmt.setString(1, email);
+			stmt.setString(2, password);
+			rs = stmt.executeQuery();
+			if(rs != null && rs.next()){
+				return getEntityFromResultSet(rs);
```

Comment:
If this method is for joining tables then getEntityFromResultSet(rs); is not enough